pkg/cortex/modules.go

```diff
@@ -397,7 +413,9 @@ func (t *Cortex) initDeleteRequestsStore() (serv services.Service, err error) {
 	return
 }
 
-func (t *Cortex) initQueryFrontend() (serv services.Service, err error) {
+// initTripperware instantiates the tripperware used by the query frontend
+// to optimize Prometheus query requests.
+func (t *Cortex) initTripperware() (serv services.Service, err error) {
```

Comment:
Maybe we could rename it to `QueryFrontTripperware()` for clarify?

Reply:
Makes sense